docs/source/markdown/podman-import.1.md

```diff
@@ -19,6 +19,10 @@ Note: `:` is a restricted character and cannot be part of the file name.
 
 ## OPTIONS
 
+#### **--arch**
+
+Set architecture of the imported image.
```

Comment:
are there valid values we can list?

Reply:
@TomSweeneyRedHat User is free to set any value if he is doing it manually however it's user's responsibility to make it work if they choose a wrong value.